messages/context.json

```diff
@@ -149,5 +149,10 @@
   "admin/editor.skuSelector.title": "Title of SKU Selector component",
   "admin/editor.skuSelector.description": "Description of SKU Selector component",
   "admin/editor.skuSelector.seeMoreLabel.title": "Title of seeMoreLabel property",
-  "admin/editor.skuSelector.seeMoreLabel.description": "Description of seeMoreLabel property"
+  "admin/editor.skuSelector.seeMoreLabel.description": "Description of seeMoreLabel property",
+  "vtex.native-types::store/native-types.text": "native types",
+  "vtex.native-types::store/native-types.richText": "Rich text field",
+  "vtex.native-types::store/native-types.url": "URL field",
+  "vtex.address-form::address-form.field.postalCode": "address-form.field.postalCode",
```

Comment:
Let's discuss this. 

Store-components it's not using this message implicitly. Store-components it's using a component from the vtex.address-form app and this component it's using its own message. 

The point is, this message is an implementation detail of the app address-form.